expyfun/_sound_controllers.py

```diff
@@ -5,10 +5,12 @@
 #
 # License: BSD (3-clause)
 
-import numpy as np
-from scipy import fftpack
 import sys
 import os
+import warnings
+
+import numpy as np
+
```

Comment:
It's a convention borrowed from MNE to separate standard imports, then standard scientific library imports, then local imports